app/helpers/application_helper.rb

```diff
@@ -766,6 +766,25 @@ def payment_settings_url(gateway_type, person, url_params)
     end
   end
 
+  def external_plan_service_login_url(marketplace_id)
+    payload = {user_id: marketplace_id}
+    secret = APP_CONFIG.external_plan_service_secret
+    external_plan_service_url = APP_CONFIG.external_plan_service_url + "login"
+    token = JWTUtils.encode(payload, secret)
+    URLUtils.append_query_param(external_plan_service_url, "token", token)
+  end
+
+  def display_expiration_notice?
+    if feature_enabled?(:new_plan_page)
+      display = APP_CONFIG.external_plan_service_in_use && 
+        @current_user && @current_user.has_admin_rights_in?(@current_community) &&
```

Comment:
Instead of 

```
@current_user && @current_user.has_admin_rights_in?(@current_community)
```

You can also use `Maybe` here:

```
Maybe(@current_user).has_admin_rights_in?(@current_community).or_else(false)
```

Matter of taste.